movie/movie.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../item'
+
+class Movie < Item
+  attr_accessor :silet
+
+  def initialize publish_date, silet
```

Comment:
- [ ] Hello, Kindly use parentheses for this constructor so that the Rubocop offense found is fixed.
 - Kindly change it to `  def initialize (publish_date, silet)`